src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -260,12 +260,15 @@ public AssumeRoleRequest createAssumeRoleRequest(Claims claims, String roleArn,
 		
 		for (String claimName: getTagClaimNames()) {
 			Object claimValue = claims.get(claimName);
-			if (claimValue!=null) {
-				sessionTags.put(TAG_PREFIX+claimName, claimValue.toString());
+			if (TEAM_CLAIM_NAME.equals(claimName)) {
```

Comment:
if i understand correctly this only happens when `SESSION_TAG_CLAIMS` contains `team`?  

You've removed `team` from the properties files so when would this condition ever be true?

Reply:
> when would this condition ever be true?

It will be true if 'team' is ever added to the configuration file.

Previously 'team' was added as a tag regardless of the configuration.  Now the configuration controls whether 'team' is added.  

For now we omit 'team' from the configuration since it causes the problem seen in SC-178.  We hope eventually to get to the bottom of the issue and be able to reenable 'team' as a session tag.